src/Libraries/Microsoft.Extensions.AI.Evaluation.Reporting/CSharp/ScenarioRunResult.cs

```diff
@@ -81,6 +83,11 @@ public ScenarioRunResult(
     {
     }
 
+    /// <summary>
+    /// Gets or sets the Version of the format used to persist this scenario run. You should not modify this value from the default.
```

Comment:
> You should not modify this value from the default.

Then why is it settable?

Reply:
The reporting types are all completely mutable - we made this call earlier during API reviews as that just makes things simpler for serialization (and to avoid the need for special constructors etc.). We also decided to keep things simple and prefer `set` over `init` in all cases. I don't think we should revisit that call or deviate from it solely for this property :)

Reply:
Then why say it shouldn't be changed?

The comment is just at odds with it being mutable.

Reply:
Ah good catch - sorry I missed the quoted text in your original comment. I agree, I think we should remove the guidance.